airbyte-cdk/python/airbyte_cdk/sources/declarative/retrievers/simple_retriever.py

```diff
@@ -394,6 +394,13 @@ def _parse_records(
     def must_deduplicate_query_params(self) -> bool:
         return True
 
+    @property
+    def max_retries(self) -> Union[int, None]:
```

Comment:
I'm confused. This only usage I see of this method is in the tests. Can you point out where else it is used?